src/components/contact/ContactCard.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import TwitterIcon from "@/assets/icons/twitter.svg?component";
+import FacebookIcon from "@/assets/icons/facebook.svg?component";
+import LinkedinIcon from "@/assets/icons/linkedin.svg?component";
+import InstagramIcon from "@/assets/icons/instagram.svg?component";
+import GithubIcon from "@/assets/icons/github.svg?component";
+import BehanceIcon from "@/assets/icons/behance.svg?component";
+import DribbleIcon from "@/assets/icons/dribble.svg?component";
+import PinIcon from "@/assets/icons/pin.svg?component";
+import EmailIcon from "@/assets/icons/email.svg?component";
+import PhoneIcon from "@/assets/icons/phone.svg?component";
+</script>
+
+<template>
+  <div class="contact-card">
+    <div class="contact-card__map-container">
+      <img
+        src="@/assets/images/map.png"
+        alt="Map of Monterail office location Wroclaw City"
+      />
+    </div>
+    <div class="contact-card__address">
+      <div class="contact-card__contact-icon">
+        <PinIcon />
+      </div>
+      <p>
+        Monterail Sp. z o.o. <br />
+        ul. Oławska 27-29 <br />
+        50-123 Wrocław<br />
+        Poland
+      </p>
+      <div class="contact-card__contact-icon">
+        <EmailIcon />
+      </div>
+      <p>hello@monterail.com</p>
+      <div class="contact-card__contact-icon">
+        <PhoneIcon />
+      </div>
+      <p>+48 533 600 136</p>
+    </div>
+    <div class="contact-card__social">
+      <router-link to="#" class="contact-card__icon">
+        <TwitterIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <FacebookIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <LinkedinIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <InstagramIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <GithubIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <BehanceIcon />
+      </router-link>
+      <router-link to="#" class="contact-card__icon">
+        <DribbleIcon />
+      </router-link>
```

Comment:
external links (so links that are redirecting outside of your domain should probably not be a `<router-link>`s but rather plain `<a>` tags.